*** ./build/test/lxgw-fonts.luatex.tlg	2026-05-14 11:52:48.294144885 +0000
--- ./build/test/lxgw-fonts.luatex.log	2026-05-14 11:53:02.130656872 +0000
***************
*** 12,25 ****
  (fontspec)             - 'bold' (b/n) with NFSS spec.: <->"[LXGWNeoZhiSong.ttf]:mode=node;script=latn;language=dflt;jfm=zh_CN/quanjiao;embolden=1.5;"
  (fontspec)             - 'italic' (m/it) with NFSS spec.: <->"[LXGWNeoZhiSong.ttf]:mode=node;script=latn;language=dflt;jfm=zh_CN/quanjiao;slant=0.2;"
  (fontspec)             - 'bold italic' (b/it) with NFSS spec.: <->"[LXGWNeoZhiSong.ttf]:mode=node;script=latn;language=dflt;jfm=zh_CN/quanjiao;embolden=1.5;slant=0.2;"
! (load cache: extra_lmroman12-regular.luc)
  Underfull \hbox (badness 10000) detected at line 51
  [][]
  \hbox(12.672+1.728)x14.4, direction TLT
  .\whatsit4=[]
  .\hbox(12.672+1.728)x7.2, direction TLT
  ..\LTJY3/LXGWNeoZhiSong(1)/m/n/14.4 。
! (load cache: extra_lmroman12-bold.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/n' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/b/n' tried instead on input line ....
  Underfull \hbox (badness 10000) detected at line 51
--- 12,25 ----
  (fontspec)             - 'bold' (b/n) with NFSS spec.: <->"[LXGWNeoZhiSong.ttf]:mode=node;script=latn;language=dflt;jfm=zh_CN/quanjiao;embolden=1.5;"
  (fontspec)             - 'italic' (m/it) with NFSS spec.: <->"[LXGWNeoZhiSong.ttf]:mode=node;script=latn;language=dflt;jfm=zh_CN/quanjiao;slant=0.2;"
  (fontspec)             - 'bold italic' (b/it) with NFSS spec.: <->"[LXGWNeoZhiSong.ttf]:mode=node;script=latn;language=dflt;jfm=zh_CN/quanjiao;embolden=1.5;slant=0.2;"
! (load cache: ../extra_lmroman12-regular.lua.gz)(load cache: ../extra_lmroman12-regular.luc)
  Underfull \hbox (badness 10000) detected at line 51
  [][]
  \hbox(12.672+1.728)x14.4, direction TLT
  .\whatsit4=[]
  .\hbox(12.672+1.728)x7.2, direction TLT
  ..\LTJY3/LXGWNeoZhiSong(1)/m/n/14.4 。
! (load cache: ../extra_lmroman12-bold.lua.gz)(load cache: ../extra_lmroman12-bold.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/n' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/b/n' tried instead on input line ....
  Underfull \hbox (badness 10000) detected at line 51
***************
*** 62,75 ****
  ============================================================
  TEST 2: Basic Usage of LXGW Font: \itshape \songti 
  ============================================================
! (load cache: extra_lmroman10-italic.luc)(load cache: extra_lmroman12-italic.luc)
  Underfull \hbox (badness 10000) detected at line 51
  [][]
  \hbox(12.672+1.728)x14.4, direction TLT
  .\whatsit4=[]
  .\hbox(12.672+1.728)x7.2, direction TLT
  ..\LTJY3/LXGWNeoZhiSong(1)/m/it/14.4 。
! (load cache: extra_lmroman10-bolditalic.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/it' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/b/it' tried instead on input line ....
  Underfull \hbox (badness 10000) detected at line 51
--- 62,75 ----
  ============================================================
  TEST 2: Basic Usage of LXGW Font: \itshape \songti 
  ============================================================
! (load cache: ../extra_lmroman10-italic.lua.gz)(load cache: ../extra_lmroman10-italic.luc)(load cache: ../extra_lmroman12-italic.lua.gz)(load cache: ../extra_lmroman12-italic.luc)
  Underfull \hbox (badness 10000) detected at line 51
  [][]
  \hbox(12.672+1.728)x14.4, direction TLT
  .\whatsit4=[]
  .\hbox(12.672+1.728)x7.2, direction TLT
  ..\LTJY3/LXGWNeoZhiSong(1)/m/it/14.4 。
! (load cache: ../extra_lmroman10-bolditalic.lua.gz)(load cache: ../extra_lmroman10-bolditalic.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/it' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/b/it' tried instead on input line ....
  Underfull \hbox (badness 10000) detected at line 51
***************
*** 112,123 ****
  ============================================================
  TEST 3: Basic Usage of LXGW Font: \slshape \songti 
  ============================================================
! (load cache: extra_lmromanslant10-regular.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(0)/m/sl' in size <10> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(0)/m/it' tried instead on input line ....
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/m/sl' in size <10> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/m/it' tried instead on input line ....
! (load cache: extra_lmromanslant12-regular.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/m/sl' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/m/it' tried instead on input line ....
  Underfull \hbox (badness 10000) detected at line 51
--- 112,123 ----
  ============================================================
  TEST 3: Basic Usage of LXGW Font: \slshape \songti 
  ============================================================
! (load cache: ../extra_lmromanslant10-regular.lua.gz)(load cache: ../extra_lmromanslant10-regular.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(0)/m/sl' in size <10> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(0)/m/it' tried instead on input line ....
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/m/sl' in size <10> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/m/it' tried instead on input line ....
! (load cache: ../extra_lmromanslant12-regular.lua.gz)(load cache: ../extra_lmromanslant12-regular.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/m/sl' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/m/it' tried instead on input line ....
  Underfull \hbox (badness 10000) detected at line 51
***************
*** 126,132 ****
  .\whatsit4=[]
  .\hbox(12.672+1.728)x7.2, direction TLT
  ..\LTJY3/LXGWNeoZhiSong(1)/m/sl/14.4 。
! (load cache: extra_lmromanslant10-bold.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/sl' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/it' tried instead on input line ....
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/it' in size <14.4> not available
--- 126,132 ----
  .\whatsit4=[]
  .\hbox(12.672+1.728)x7.2, direction TLT
  ..\LTJY3/LXGWNeoZhiSong(1)/m/sl/14.4 。
! (load cache: ../extra_lmromanslant10-bold.lua.gz)(load cache: ../extra_lmromanslant10-bold.luc)
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/sl' in size <14.4> not available
  (Font)              Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/it' tried instead on input line ....
  LaTeX Font Info:    Font shape `LTJY3/LXGWNeoZhiSong(1)/bx/it' in size <14.4> not available
***************
*** 171,177 ****
  ============================================================
  TEST 4: Basic Usage of LXGW Font: \heiti 
  ============================================================
! (load cache: extra_lxgwneoxihei.luc)(load cache: extra_lxgwneoxihei.lua.gz)(load cache: ../extra_lxgwneoxihei.lua.gz)(load cache: ../extra_lxgwneoxihei.luc)
  Package fontspec Info: 
  (fontspec)             Font family 'LXGWNeoXiHei(0)' created for font 'LXGWNeoXiHei' with options [NFSSEncoding=LTJY3,JFM=zh_CN/quanjiao,Extension={.ttf},AutoFakeBold,AutoFakeSlant,LTJFONTUID=5].
  (fontspec)              
--- 171,177 ----
  ============================================================
  TEST 4: Basic Usage of LXGW Font: \heiti 
  ============================================================
! (load cache: ../extra_lxgwneoxihei.lua.gz)(load cache: ../extra_lxgwneoxihei.luc)
  Package fontspec Info: 
  (fontspec)             Font family 'LXGWNeoXiHei(0)' created for font 'LXGWNeoXiHei' with options [NFSSEncoding=LTJY3,JFM=zh_CN/quanjiao,Extension={.ttf},AutoFakeBold,AutoFakeSlant,LTJFONTUID=5].
  (fontspec)              
***************
*** 331,337 ****
  ============================================================
  TEST 7: Basic Usage of LXGW Font: \fangsong 
  ============================================================
! (load cache: extra_zhuquefangsong-regular.luc)(load cache: extra_zhuquefangsong-regular.lua.gz)(load cache: ../extra_zhuquefangsong-regular.lua.gz)(load cache: ../extra_zhuquefangsong-regular.luc)
  Package fontspec Info: 
  (fontspec)             Font family 'LXGWZhuqueFangsong-Regular(0)' created for font 'LXGWZhuqueFangsong-Regular' with options [NFSSEncoding=LTJY3,JFM=zh_CN/quanjiao,Extension={.ttf},AutoFakeBold,AutoFakeSlant,LTJFONTUID=6].
  (fontspec)              
--- 331,337 ----
  ============================================================
  TEST 7: Basic Usage of LXGW Font: \fangsong 
  ============================================================
! (load cache: ../extra_zhuquefangsong-regular.lua.gz)(load cache: ../extra_zhuquefangsong-regular.luc)
  Package fontspec Info: 
  (fontspec)             Font family 'LXGWZhuqueFangsong-Regular(0)' created for font 'LXGWZhuqueFangsong-Regular' with options [NFSSEncoding=LTJY3,JFM=zh_CN/quanjiao,Extension={.ttf},AutoFakeBold,AutoFakeSlant,LTJFONTUID=6].
  (fontspec)              
